src/OpenLoco/Windows/TerraForm.cpp

```diff
@@ -870,10 +869,10 @@ namespace OpenLoco::Ui::Windows::Terraform
 
         static void drawTreeThumb(TreeObject* treeObj, Gfx::Context* clipped)
         {
-            uint32_t image = _byte_500775[treeObj->growth] * treeObj->num_rotations;
+            uint32_t image = treeGrowth[treeObj->growth] * treeObj->num_rotations;
```

Comment:
I think it's a bit confusing that this is basically a global that's defined in the tree object header. Perhaps it's better to expose the tree growth array through a member function?